scripts/convert_stl_to_obj.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import os
+import trimesh
+import argparse
+
+
+def convert_stl_to_obj(directory):
+    """
+    Finds all STL files in the given directory and subdirectories, and converts each to an OBJ file.
+
+    Args:
+        directory (str): The path to the directory to search.
+    """
+    for root, _, files in os.walk(directory):
+        for file in files:
+            if file.lower().endswith(".stl"):
```

Comment:
Since you're already calling `os.path.splitext()` later, I believe the second element of that will already give you the extension. So you don't need to additionally do an `endswith()`